.github/workflows/codeql.yml

```diff
@@ -91,11 +91,17 @@ jobs:
             libzstd-dev \
             unixodbc-dev
 
+    - name: Install latest ninja
+      run: pip install ninja
+
     - name: Configure
       if: matrix.language == 'c-cpp'
       run: |
-          mkdir build
-          (cd build && cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -DGDAL_USE_LERC_INTERNAL=OFF)
+          cmake -S . -B build \
+            -DCMAKE_BUILD_TYPE=Debug \
+            -DGDAL_USE_LERC_INTERNAL=OFF \
+            -DBUILD_TESTING=OFF \
+            -G Ninja \
```

Comment:
This is valid syntax, and I deliberately left the trailing line continuation so that adding/removing cmake flags will not have to change two lines, the diffs will be smaller